package.json

```diff
@@ -366,6 +366,11 @@
           "type": "boolean",
           "default": true,
           "description": "Specifes whether OmniSharp should use VS Code editor settings for C# code formatting (use of tabs, indentation size)."
+        },
+        "omnisharp.format.enable": {
+          "type": "boolean",
+          "default": true,
+          "description": "Specifes whether OmniSharp should be used for C# code formatting."
```

Comment:
Yes, many probably should -- at least those that require OmniSharp to be relaunched (which technically doesn't require VS Core to be relaunched).

However, there are other settings like "csharp.suppressDotnetInstallWarning", "csharp.unitTestDebuggingOptions", "csharp.suppressDotnetRestoreNotification" and "csharp.suppressHiddenDiagnostics" that do not require a restart.